backend/deepchecks_monitoring/schema_models/alert_webhook.py

```diff
@@ -270,18 +270,19 @@ def prepare_request_parameters(self, alert: "Alert", settings: "Settings") -> t.
                 "headers": self.http_headers,
                 "json": {
                     "payload": {
-                        "summary": f"New {alert_rule.alert_severity.value} alert: {monitor.name}",
+                        "summary": f"New monitor alert: {monitor.name}",
                         "timestamp": pdl.instance(t.cast(datetime, alert.created_at)).to_iso8601_string(),
-                        "source": f"models/{model.name}/monitors/{monitor.name}",
+                        "source": f"models: {model.name} monitors: {monitor.name}",
                         "severity": severity,
                         "component": "deepchecks",
                         "group": additional_arguments.get("group") or "deepchecks",
                         "class": additional_arguments.get("class"),
                         "custom_details": {
-                            "deepchecks_alert_id": alert.id,
-                            "deepchecks_alert_start_time": str(alert.start_time),
-                            "deepchecks_alert_end_time": str(alert.end_time),
-                            "deepchecks_alert_failed_values": alert.failed_values,
+                            "alert_id": alert.id,
+                            "alert_name": f"models/{model.name}/monitors/{monitor.name}",
```

Comment:
maybe alert_link?

Reply:
no, it is not a link
it also should be `f"model: {model.name} monitor: {monitor.name}",`